content/platform/data-services-cloud-console-api/home.md

```diff
@@ -0,0 +1,133 @@
+
+Some of the advantages to distribution in OpenAPI 3.0 format:
+
+1. Updates to the API can be generated in more agile manner where documentation are embedded as description to any endpoints, parameters, and many more such as contact information, license, terms of use.
+2. Consumer of this API can also consume this API in an agile manner using the converter from this openAPI format to any language that were used as part of the their automation or CI/CD workflow. (Please check <https://openapi.tools> for more information for the API tools to generate client code)
+
+## REST API Details
+
+Any customer can download this OpenAPI Specification (OAS) v3 definition of the DSCC from the following link:
+
+[Link to the API repository](https://docs.ccs.arubathena.com/)
+
+### Documentation and Tutorials
+
+The overview of the Data Services Cloud Console is available at [this link](https://www.hpe.com/us/en/storage/data-services-cloud-console.html#overview). There will be more blogs that will provide more information to help customer adopting this API with examples, code snippets and many other helpful information.
+
+### Documented Attributes
+
+* **API Name & Description** - Provide short description of the goal for this API with the supported HTTP request method (POST, GET, etc)
+* **API Path** - Provide detail URL path as the end-point to issue the API call
+* **API Parameter**s - 
+
+### Supported API Categories (Services)
+
+The API categories for DSCC grows in accordance to the expansion of the DSCC services that are going to released in future. Current services that are available as recorded today is:
+
+1. DataOps Manager
+2. Event Audit
+3. Task tracking
+4. Search
+5. Helps, Issues and announcement
+
+### Versioning
+
+The major version number will be provided in the resource path as v1 in this example:
+
+```md
+/api/v1/<resource group>/...
+```
+
+Some examples of these resource paths:
+
+```markdown
+/api/v1/storage-systems/...
+
+/api/v1/controllers/...
+
+/api/v1/volumes/...
+```
+
+Some examples of the resource groups under the same root:
+
+Clients will be able to adopt the backward compatibility from the higher major version incremental.  However, both the new and old version of API will be supported until the announcement of the deprecation. Nonetheless, the older major version will always be frozen with exception of bug fixes. There will also be announcement of deprecation in the header and sunset header. 
```

Comment:
Consider rephrasing the first sentence to "Clients will be able to adopt the backward capability incrementally from a higher major version."

Add the word "an" between "be" and "announcement" in the last sentence.